app/assets/scripts/components/bar-chart-measurment.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { Bar } from 'react-chartjs-2';
+
+export default function BarChart({ title, frequency, xAxisLabels }) {
+  const data = {
+    labels: xAxisLabels,
+    datasets: [
+      {
+        label: 'measurments',
+        data: frequency,
+        backgroundColor: '#198CFF',
+      },
+    ],
+  };
+
+  const options = {
+    scales: {
```

Comment:
I think we can hide the legend, like this:
```suggestion
    legend: {
      display: false,
    },
    scales: {
```